_scripts/startTest.sh

```diff
@@ -0,0 +1,2 @@
+docker-compose -f docker-compose.yaml -f docker-compose.test.yaml up -d
+docker exec -it demo_test_api bash -c "npm install && npm test"
```

Comment:
We may be making a mistake, but when we try to run the test script, we get a daemon error. But we don't know why